generator/metricbeat/Makefile

```diff
@@ -1,9 +1,16 @@
 BEAT_TYPE=metricbeat
-PREPARE_COMMAND=MODULE=elastic METRICSET=test make create-metricset ;
+PREPARE_COMMAND=MODULE=elastic METRICSET=test make create-metricset ; FIELDS_FILE_PATH=module
+override FIELDS_FILE_PATH=
```

Comment:
What are these 2 lines exactly doing?

Reply:
This overrides `FIELDS_FILE_PATH` coming from the Makefile of Metricbeat. The problem is that the variable is already set, and the only way I have found to override it is this kind of "ugly" way.

Reply:
As auditbeat depends on Metricbeat, would it then be need there too?

Reply:
Yes, but only if the `FIELDS_FILE_PATH` is different from Metricbeat. But both Beats store their modules under `module`, so there is no need to add that variable to the Makefile of Auditbeat yet.